lib_comfyui/comfyui_adapter.py

```diff
@@ -19,18 +20,23 @@ def unwrap_cpu_state_dict(state_dict: dict) -> dict:
     }
 
 
+def get_opts_outdirs():
+    return json.loads(shared.opts.dumpjson())

```

Comment:
The reason this didn't work might be that you want to serialize `shared.opts.data`, not just `shared.opts`.

Reply:
For reference, this is how the webui creates the dict before serializing it
```python
{k: self.data.get(k, v.default) for k, v in self.data_labels.items()}
```

Reply:
Something else we could do would be to json serialize here and deserialize in the comfyui process.